test/clj_sqs_extended/core_test.clj

```diff
@@ -122,6 +124,21 @@
           (is (contains? stats :stopped-at))))
       (close! handler-chan))))
 
+(deftest handle-queue-terminates-with-non-existing-bucket
```

Comment:
Related case. Do we have a test to check that `handle-queue` works if s3-bucket-name is nil? That's what I've integrated with so I know it works. But it would be good to have an explicit test.

Reply:
Yes, good idea. I'll add it.